content/en/docs/contribute/new-content/overview.md

```diff
@@ -54,5 +54,8 @@ was wrong, you (and only you, the submitter) can change it.
 
 Limit pull requests to one language per PR. If you need to make an identical change to the same code sample in multiple languages, open a separate PR for each language.
 
+## Tools for contributors
+
+The [doc contributors tools](https://github.com/kubernetes/website/tree/master/content/en/docs/doc-contributor-tools) directory in the `kubernetes/website` repository contains tools to help your contribution journey go more smoothly.
```

Comment:
@celestehorgan , I wonder if  `docs/doc-contributor-tools` is a good location for this information.
Does anyone know that section exists?

Reply:
@kbhawkey I thought about that as well. I think in terms of co-locating similar content (fun tools to make contributor life easier) it's in the right place, but I'm also against random directories in the `content/en/docs` with random things in them that aren't content.

In the end: I decided it was best not to push through a reorganization of contributor docs in this PR. ;)